Some machines need more than 5 minutes to run some tests

Hello. I tried to build the Debian package for this project on AWS instances of type `r7a.medium`, which incidentally have a single CPU, and two tests failed because 300 seconds is not enough to reach an optimal solution.

The errors are like this:

```
[ERROR] Failures: 
[ERROR]   CuteMarosMeszarosCase.testMOSARQP2:466->doTest:233->doTest:226 FEASIBLE -742.6974600044708 @ { 0.89737817719434, [ lot-of-stuff ]  0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0 } ==> expected: <true> but was: <false>
[ERROR]   MIPLIBTheEasySet.testPk1:393->doTest:63 FEASIBLE 29.99999999999989 @ { 29.99999999999989, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 1, 1, 0, 1, 1, 0, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 1, 0, 0, 0, 1, 1, 0, 1, 0, 29.99999999999997, 18.99999999999975, 19.99999999999982, 0, 29.99999999999979, 5.00000000000004, 1.9999999999997, 0, 15.99999999999975, 0, 0.99999999999991, 0, 18.99999999999984, 29.99999999999988, 0, 5.00000000000016, 3.99999999999955, 0, 21, 0, 0, 29.9999999999999, 17.00000000000015, 29.99999999999987, 0, 1.00000000000052, 0, 11.99999999999966, 0, 23.00000000000018 } ==> expected: <true> but was: <false>
```

The AWS instances are not really "slow", they have 4th generation EPYC processors from AMD, but 300 seconds is not enough for those tests to pass.

To fix this, I suggest increasing `time_suffice` in ModelFileTest.java from 5 minutes to 10 minutes.

This way, machines reaching optimal solutions in less than 5 minutes will continue to do so after such change and will not be affected, but some machines requiring a little bit more than 5 minutes will finally be able to pass all the tests.

For completeness, I'm opening a Pull Request doing exactly that, even if it's a one-line change.

An easy way to reproduce this (other than actually using a machine of type `r7a.medium` from AWS) is to configure GRUB with `GRUB_CMDLINE_LINUX="nr_cpus=1"` and rebooting afterwards.

Thanks.

--- a/src/test/java/org/ojalgo/optimisation/ModelFileTest.java
+++ b/src/test/java/org/ojalgo/optimisation/ModelFileTest.java
@@ -133,7 +133,7 @@ public interface ModelFileTest {
                 }
             }
 
-            model.options.time_suffice = 5L * CalendarDateUnit.MINUTE.toDurationInMillis();
+            model.options.time_suffice = 10L * CalendarDateUnit.MINUTE.toDurationInMillis();
             model.options.time_abort = 15L * CalendarDateUnit.MINUTE.toDurationInMillis();
 
             TestUtils.assertTrue(model.validate());

